src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -867,7 +867,7 @@ def main():
     if showtiming:
         print("... in %s seconds." % (time.time() - start_time))
 
-    if "parity_check_input_folder" in parity_parameters:
+    if "parity_check_input_folder" in parity_parameters or "parity_check_file" in parity_parameters:
```

Comment:
To allow for parity assessment based on validation data stored in .csv, I appended this if statement to work if either "parity_check_input_folder" or "parity_check_file" is in paraity_parameters

Reply:
Might consider a check on the length of parity_parameters...